### ISSUE TITLE
the dial button on the RTO pack doesnt do anything

### ISSUE THREAD

**Issue by Dutch-VanDerLinde** [RMC-14/RMC-14#4825]
## Description

**Reproduction**

**Screenshots**

**Additional context**

**Comment by Skarletto** [comment]
The intended behavior should be to hide or obscure the dial button until a name is highlighted, THEN you can dial. Currently you only click on the person's name to call them.

![image](https://github.com/user-attachments/assets/c5501fb1-ab5e-4ece-9894-ead25cddf9b6)
![image](https://github.com/user-attachments/assets/8cb0e275-dcf0-49ed-b9d8-b7e281446f3f)
